prediction_market_agent_tooling/markets/agent_market.py

```diff
@@ -43,6 +43,14 @@ class AgentMarket(BaseModel):
     def p_no(self) -> float:
         return 1 - self.p_yes
 
+    @property
+    def boolean_outcome(self) -> bool:
+        if not self.has_successful_resolution():
+            raise ValueError(
+                "Market must have successful resolution to compute boolean outcome."
+            )
+        return self.resolution == Resolution.YES
```

Comment:
```suggestion
        return True if self.resolution == Resolution.YES else False if self.resolution == Resolution.NO else should_not_happen(f"Probably a bug in self.has_successful_resolution")
```

Just to prevent any unnoticed bugs 🙏